src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class Circle extends Figure {
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+    private double radius;
+    @Override
+    public String toString() {
+        return "Circle{" +
+                "radius=" + radius +
+                '}';
+    }
+    @Override
+    public double area() {
+        return Math.PI * radius * radius;
```

Comment:
All magic numbers in your code should be constants. The value of `Math.PI` is okay, but if you had other numbers, remember to replace them with named constants for better code readability and maintainability.